### ISSUE TITLE
When in a space i can only see DM from people in the space

### ISSUE THREAD

**Issue by MarcWadai** [tchapgouv/tchap-web-v4#1019]
Product ticket https://github.com/tchapgouv/tchap-product/issues/128

## Description 
![image](https://github.com/tchapgouv/tchap-web-v4/assets/10048869/aca86bc0-929d-4c6b-9543-d8084256efaa)

When this option is activated and that im in a space i will be able to see the DMs only with people that are also in the space. 
This can be confusing when you have lot of spaces and DMs with people that also share some spaces but not all.

## Possible solution
Maybe changing the title of the DM section in a space. Instead of : 

Messages directs -> Messages directs avec les membres de l'espaces (maybe too long..)

**Comment by jdauphant** [comment]
An another solution is to remove this option or disabled by default (most users never check the option and use default settings)

**Comment by MarcWadai** [comment]
Yes that's a good idea to disable by default ! Keeping the possibilities to activate will make it conscious to the users about what they see in the directs messages section. @areox-net @odelcroi what do you think ?

**Comment by areox-net** [comment]
@MarcWadai As a user I see spaces as a way of filtering my conversation. I find it strange (and useless) that in a space I can message people who don't belong to that space. 
To me, it'd make more sense to enable the feature for good and hide the setting.
